frontend/src/plugins/veeValidate.ts

```diff
@@ -13,6 +13,11 @@ extend('required', {
   message: '{_field_} can not be empty',
```

Comment:
Why you don't use this one 
Looks like it has the same logic for required field

Reply:
https://prnt.sc/26wt820 because of the ugly error message. Might be better to use this message "This field can not be empty" ?

Reply:
Delete additional rule for password.